### ISSUE TITLE
Call to undefined method OC_Util::isWebDAVWorking() at settings/admin.php#44

### ISSUE THREAD

**Issue by bitsandnumbers** [owncloud/core#13807]
Hi,

I don't really know since when, but I can't access my administration page. Each time I get a blank page and this error in the logs: 

`Call to undefined method OC_Util::isWebDAVWorking() at settings/admin.php#44`

If I comment line 44 of settings/admin.php, I can access it again. In both cases my webdav work, but I'm afraid it break something else. I wondered if I could just comment out this line 44 without issue ? Also, has someone any idea why I get this error and how I can get rid of it ?

I'm using OC 7.0.4 on a freeBSD system, with nginx and PHP 55 + extensions.

**Comment by RealRancor** [comment]
Hi,

are you sure you're running OC 7.0.4? This call doesn't exist in OC 7.0.4 at this line:

https://github.com/owncloud/core/blob/v7.0.4/settings/admin.php#L44

**Comment by MorrisJobke** [comment]
This was added between 7.0.2 and 7.0.3 (so it is in 7.0.3) PR is here: https://github.com/owncloud/core/pull/7051#issuecomment-56533466 

Commit on stable7: 2013404e0a5ccb4d0fce73ee3ff6af2ece0a1c70

**Comment by MorrisJobke** [comment]
@bitsandnumbers Can you compare your admin.php with https://github.com/owncloud/core/blob/v7.0.4/settings/admin.php ? Because this is how it should look like.

How do you installed it? Maybe your source is somehow broken?

**Comment by bitsandnumbers** [comment]
Ok, so following your statement that the call should not exist in 7.0.4 (and as I was sure to be on 7.0.4)  I checked my OC installation and re-uploaded the latest OC files *after* deleting the old files, and it appear that you were right. During the last upgrade, something must have gone wrong and some files may not have been replaced.

So it works now (except that I get this new error `Sabre\\DAV\\Exception\\NotAuthenticated: No basic authentication headers were found`that I don't know where it came from :-) ). Sorry for the false alarm.
